features/Steps/UtilitySteps.php

```diff
@@ -37,4 +39,44 @@ public function iWaitSeconds(int $seconds): void
     {
         sleep($seconds);
     }
+
+    /**
+     * @Given /^I prevent duplicate creation$/
+     */
+    public function iPreventDuplicateCreation(): void
+    {
+        $configFile = file_get_contents('config.php');
+
+        if (str_contains($configFile, "'prevent_duplicate_creation' => true")) {
+            // The config was already on true, so no futher changes are required
+            $this->initialPreventDuplicateCreationValue = true;
+            return;
+        }
+
+        $configFile = str_replace(
+            "'prevent_duplicate_creation' => false",
+            "'prevent_duplicate_creation' => true",
+            $configFile
+        );
+
+        file_put_contents('config.php', $configFile);
+
+        $this->initialPreventDuplicateCreationValue = false;
+    }
+
+    /**
+     * @Then /^I allow duplicate creation$/
+     */
+    public function iAllowDuplicateCreation(): void
```

Comment:
This method name is not correct, it does restore the prevent duplication config.